footer.php

```diff
@@ -27,7 +27,7 @@
 					<?php
 					else :
 						$blog_info = get_bloginfo( 'name' );
-						if ( ! empty( $blog_info ) ) :
+						if ( ! empty( $blog_info ) && get_theme_mod( 'display_title_and_tagline', true ) === true ) :
```

Comment:
We don't need to check if `===true`, we can simplify this check as per the suggestion below
```suggestion
						if ( ! empty( $blog_info ) && get_theme_mod( 'display_title_and_tagline', true ) ) :
```